torch/csrc/jit/tensorexpr/types.h

```diff
@@ -80,7 +78,7 @@ class TORCH_API Dtype {
       std::ostream& stream,
       const Dtype& dtype);
   ScalarType scalar_type_;
-  int lanes_; // the width of the element for a vector time
+  int64_t lanes_; // the width of the element for a vector time
```

Comment:
Does this need to be int64_t?

Reply:
Otherwise some functions for computing lanes return int64_t